RELEASE-NOTES.md

```diff
@@ -1,5 +1,48 @@
 # Prefect Release Notes
 
+## Release 2.14.16
+
+### Enhancements
+- Add the `wait_for_flow_run` method to `PrefectClient` to allow waiting for a flow run to complete — https://github.com/PrefectHQ/prefect/pull/11305
+- Add a provisioner for `Modal` push work pools — https://github.com/PrefectHQ/prefect/pull/11665
+- Add a workflow to publish OSS UI components automatically on OSS releases — https://github.com/PrefectHQ/prefect/pull/11655
+- Expose the `limit` kwarg in `serve` — https://github.com/PrefectHQ/prefect/pull/11645
+- Add methods supporting modification and suppression of flow run notification policies — https://github.com/PrefectHQ/prefect/pull/11163
+- Enhancements to sending and receiving flow run inputs — https://github.com/PrefectHQ/prefect/pull/11636
+
+
+### Fixes
+- Fix block reference resolution in `prefect.yaml` and support accessing block fields using keypaths — https://github.com/PrefectHQ/prefect/pull/10938
+- Avoid rerunning task runs forced to Completed state — https://github.com/PrefectHQ/prefect/pull/11385
+- Add a new UI setting to customize the served static directory — https://github.com/PrefectHQ/prefect/pull/11648
```

Comment:
```suggestion
- Add a new UI setting to customizE served static directory — https://github.com/PrefectHQ/prefect/pull/11648
```

Reply:
I feel like we need the "the" - maybe this is better? "Add a new UI setting to customize the location of the served static directory"